src/workerd/api/pyodide/pyodide.c++

```diff
@@ -484,6 +486,24 @@ void DiskCache::put(jsg::Lock& js, kj::String key, kj::Array<kj::byte> data) {
   }
 }
 
+jsg::JsValue SetupEmscripten::getModule(jsg::Lock& js) {
+  KJ_IF_SOME(module, emscriptenModule) {
+    return module.getHandle(js);
+  } else {
+    auto& runtime = KJ_ASSERT_NONNULL(workerd::Worker::Api::current().getEmscriptenRuntime());
```

Comment:
Well it can be guaranteed that if we have a python worker this HAS to be instantiated before we call getModule.
If for some reason someone called SetupEmscripten.getModule without an emscripten runtime this should definitely fail and there's now way to recover from it.
That being said, for any isolate that isn't a python isolate emscriptenRuntime will always be a kj::none and SetupEmscripten.getModule will simply never be called.